FAQ.md

```diff
@@ -87,29 +89,33 @@ This is set via the MCO’s config, not the Machine-api. The config a Machine re
 ## How does the user-data get created for the VMs?
 The user-data is generated by the installer.  The Machine-api is just a consumer of the user-data and simply passes the data to the cloud.  Refer to documentation for the Machine-config-operator (not part of the Machine-api) for creating new Machine pools and user-data.
 
-## Adding Labels to Nodes via Machines
-Labels are only added to Nodes via Machines at creation time.  You can add arbitrary labels which are applied to Nodes immediately after they join the cluster.  Adding subsequent labels on the Machine object (or in a MachineSet’s template) will have no effect.
+## Adding Annotations and Labels to Nodes via Machines
+Annotations and Labels are only added to Nodes via Machines at creation time.  You can add arbitrary annotations and labels which are applied to Nodes immediately after they join the cluster.  Adding subsequent annotations and labels on the Machine object (or in a MachineSet’s template) will have no effect.
 
-### Which labels get added to a Node?
-When we talk about Node labels here, it can be somewhat confusing.
+### Which Annotations and Labels get added to Nodes?
+MachineSets can be used to help mark the Machines and Nodes that are created from them by using Annotations and Labels. To do this you will need to add this information in various places on your MachineSet depending on where you would like this information. Please note that this information will only be applied to new Machines and Nodes created from the MachineSet, they will not be retroactively applied.
```

Comment:
As it's today machines reconcile towards node anytime during its lifecycle https://github.com/openshift/machine-api-operator/blob/master/pkg/controller/nodelink/nodelink_controller.go#L229-L239

The case where we only reconcile on creation is from machineSet to machines just like as any other machineSet field. Changes only affect new created replicas.

Reply:
maybe we should word this paragraph differently. i think the message we are trying to convey is that updating a machineset won't trigger a retroactive application of annotations and labels.